### DISCUSSION TITLE
Alternative to exposing CouchDB to the Internet?

### DISCUSSION THREAD

**Discussion by mermaidtrysail** [vrtmrz/obsidian-livesync#268] [Q&A]
I am currently running a CouchDB server at home and Obsidian Livesync works well.

However, I am having apprehension about a password protected database exposed to the Internet.

Is there any way to add security currently with the current version of the plug-in?

**Comment by vrtmrz** [comment]
Thank you for asking me!
I am also always worried about that. I have not checked deeply yet, how about [Cloudflare Tunnel](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/)?

**Comment by xieve** [comment]
You could set up a VPN to your home network if you have a public IP address and can forward ports. Or set up Wireguard with a cheap cloud server and your client and home server as peers.

**Reply by codedge** [reply]
If you fancy Wireguard there is Tailscale, https://tailscale.com, that builds upon than. Free of charge.. can only recommend.
